encryption_works.md

```diff
@@ -96,6 +96,32 @@ It's important to remember that just because you use free software doesn't mean
 
 Tails, which is a live DVD and live USB GNU/Linux distribution that I will discuss in detail below, solves many of these problems.
 
+## Signal and TextSecure
```

Comment:
Is this section supposed to cover VOIP as well as text messaging?  Or is the VOIP section somewhere else (and in another PR?)  If it's supposed to encompass both, please mention RedPhone for Android (and that on iOS, Signal covers both usages; while on Android you have to get 2 apps.)